doc/src/manual/faq.md

```diff
@@ -1035,17 +1035,11 @@ Modifying OpenBLAS settings or compiling Julia with a different BLAS library, eg
 
 ### How do I manage precompilation caches in distributed file systems?
 
-When using `julia` in high-performance computing (HPC) facilities, invoking
-_n_ `julia` processes simultaneously creates at most _n_ temporary copies of
-precompilation cache files. If this is an issue (slow and/or small distributed
-file system), you may:
-
-1. Use `julia` with `--compiled-modules=no` flag to turn off precompilation.
-2. Configure a private writable depot using `pushfirst!(DEPOT_PATH, private_path)`
-   where `private_path` is a path unique to this `julia` process.  This
-   can also be done by setting environment variable `JULIA_DEPOT_PATH` to
-   `$private_path:$HOME/.julia`.
-3. Create a symlink from `~/.julia/compiled` to a directory in a scratch space.
+When using Julia in high-performance computing (HPC) facilities with shared filesystems, it is recommended to use a shared
+depot (via the `JULIA_DEPOT_PATH` environment variable). Since Julia v1.10, multiple Julia processes on functionally similar
+workers and using the same depot will coordinate via pidlocks to only spend effort precompiling on one process while the
+others wait. Log messages will indicate when the process is precompiling or waiting for another that is precompiling
+(via an `@info` log if interactive or `@debug` if not)
```

Comment:
```suggestion
(via an `@info` log if interactive or `@debug` if not)

However, due to caching of binary code, the cache rejection since v1.9 is more strict, users may need to set `JULIA_CPU_TARGET=generic` environment variable to get the cache to be usable on HPC environment.
```

Reply:
Thanks. What about this?

```suggestion
(via an `@info` log if interactive or `@debug` if not)

However, due to caching of binary code, the cache rejection since v1.9 is more strict, users may need to set the
`JULIA_CPU_TARGET` environment variable to `generic`, or a narrower list of multiple values that represent the
variation in the cluster workers, to get a single cache to be usable throughout the HPC environment.
```

Reply:
What do I do when I have exactly same CPU model on login and computer node and it still doesn't work? What should I write in this case

Reply:
Maybe add that in this case it seems like a bug that should be reported? But of course, there should also be a recommendation for how to proceed anyways, I agree 👍